src/System.Globalization.Calendars/tests/project.lock.json

```diff
@@ -0,0 +1,588 @@
+{
+  "locked": false,
```

Comment:
@ericstj, I've seen this be true elsewhere... should it be changed from false here, too?